app/javascript/components/UswdsOverrides/UswdsPagination.tsx

```diff
@@ -92,7 +94,13 @@ export const UswdsPagination = ({
   const showOverflow = totalPages > maxSlots; // If more pages than slots, use overflow indicator(s)
 
   const middleSlot = Math.round(maxSlots / 2); // 4 if maxSlots is 7
-  const showPrevOverflow = showOverflow && currentPage > middleSlot;
+  let showPrevOverflow; 
+  if (!unboundedResults) {
```

Comment:
The pagination component from react-uswds doesn't support unbounded pagination, and we're using a modified version of unbounded pagination for Bing searches, so all changes in this file are for Bing pagination.